modules/alloy/src/main/ts/ephox/alloy/behaviour/tooltipping/ActiveTooltipping.ts

```diff
@@ -62,12 +62,12 @@ const events = (tooltipConfig: TooltippingConfig, state: TooltippingState): Allo
       AlloyEvents.run(ShowTooltipEvent, (comp) => {
         state.resetTimer(() => {
           show(comp);
-        }, tooltipConfig.delay);
+        }, tooltipConfig.delayForShow());
```

Comment:
The change here was to make this a function. By making it a function, we can have something know if we are in "tooltipping mode" and if we are, then just show immediately.

We also separate show and hide, because hide shouldn't care about tooltipping mode.